Gemfile

```diff
@@ -265,4 +265,4 @@ end
 
 gem 'kubeclient'
 
-gem 'pg', '~> 0.21.0'
+gem 'pg', '~> 1.3'
```

Comment:
It seems that in the latest Rails 6.1 the required version is [">= 1.3.0.rc1"](https://github.com/rails/rails/blame/v6.1.7.7/Gemfile#L137), but the error I was getting:

```
LoadError: Error loading the 'postgresql' Active Record adapter. Missing a gem it depends on? can't activate pg (~> 1.1), already activated pg-0.21.0. Make sure all dependencies are added to Gemfile.
```

Reply:
such error I usually see when I forget to call `bundle exec ...` but it's fine to get it updated either way

Reply:
Forgetting `bundle exec` was not the case, but indeed there was something weird :grimacing: 

I sometimes get this error for the default gems (like `json`).